tests/snmpsim/fs-nmu.snmprec

```diff
@@ -4,13 +4,9 @@
 1.3.6.1.2.1.1.4.0|4|<private>
 1.3.6.1.2.1.1.5.0|4|<private>
 1.3.6.1.2.1.1.6.0|4|<private>
-1.3.6.1.4.1.40989.10.16.1.2.2.0|4|OEO
```

Comment:
Hi @jonathanjdavis 
Please do not change any existing snmprec file. Add your new snmprec file for sure, and compute existing and new JSON files as well.
Thanx

Reply:
Hi @PipoCanaja , the existing fs-nmu os was updated because in the hardware, serial, and version the previous contributor had these details pulled from a module that might not be installed in that location on other extract same fs chassis. OEOs are modules that can be installed in 1 of 16 slots, with different mibs (1-16) depending on the slot the OEO(EDFA, or other modules) are installed in. The modules hardware, serial, and version were replaced with the chassis hardware serial and version, and I had to update the test data accordingly.

I'll revert and add our testing data without deleting existing.